Changelog.md

```diff
@@ -1,5 +1,79 @@
 # Changelog
 
+### 1.3.1
+
+Please fill out our 2022 Ingress-Nginx User Survey and let us know what you want to see in future releases.
+
+https://www.surveymonkey.com/r/ingressngx2022
+
+Also, *important note*, with the Release 1.25 we are dropping support for the legacy branches, 
+please upgrade to a v1.0.0 and above branch
+
+## Image:
+- registry.k8s.io/ingress-nginx/controller:v1.3.1@sha256:54f7fe2c6c5a9db9a0ebf1131797109bb7a4d91f56b9b362bde2abd237dd1974
+- registry.k8s.io/ingress-nginx/controller-chroot:v1.3.1@sha256:a8466b19c621bd550b1645e27a004a5cc85009c858a9ab19490216735ac432b1
+
+
+## What's Changed
+
+_IMPORTANT CHANGES:_
```

Comment:
We need to remind that starting from this release, elections will be done entirely using the Lease API and no longer using configmaps.
v1.3.0 is a safe transition version, using v1.3.0 can automatically complete the merging of election locks, and then you can safely upgrade to v1.3.1.
For fresh installs, there is no need to worry

Reply:
added